plugin/src/adapter/kotlin/org/jlleitschuh/gradle/ktlint/reporter/GenericReporterProvider.kt

```diff
@@ -0,0 +1,14 @@
+package org.jlleitschuh.gradle.ktlint.reporter
+
+import java.io.PrintStream
+
+/**
+ * Abstraction over ReporterProvider and ReporterProviderV2
+ */
+interface GenericReporterProvider<T : GenericReporter<*>> {
+    val id: String
+    fun get(
+        out: PrintStream,
```

Comment:
You might want to consider renaming this variable as it conflicts with kotlin's `out` keyword.

Don't feel obligated to do this, but something to consider

Reply:
good call